tests/components/spreadsheet.test.ts

```diff
@@ -419,6 +419,24 @@ describe("Composer interactions", () => {
     expect(fixture.querySelector(".o-grid .o-autocomplete-dropdown")).toBeNull();
   });
 
+  test("autocomplete disappear when starting with an unknown character", async () => {
```

Comment:
I would give a different name to these two tests, IMO it's not because you start with an unknown character, it's because there is not match with an unknown character, right ?